### ISSUE TITLE
Folders on the Homescreen does not work with Talkback

### ISSUE THREAD

**Issue by DeletedMember** [bearinmindcat/Launcher314#38]
Launcher314 v. 0.0.9 beta

I was very happy that the Launcher314 works very well with Talkback (a lot of Launchers don't do this) 

But now I have found - the Folder on the Homescreen do not work with Talkback / Android for accessibility - so they are not recognized and so you can neither open them or do anything else. For Talkback, they just don't exist. 

In the App Drawer it works without any problems. All Folders are recognized and you can use all Options.

**Comment by bearinmindcat** [comment]
I'll get a hotfix in this weekend to address this among other issues with the release. (like the version)

**Comment by ghost** [comment]
I just tested something and could find (my) Mistakes.

With the Folders on the Homescreen I had hidden the Label.
If I leave the Label displayed, the Folders also work with Talkback.
They are recognized just like in the Drawer and I can use them quite normally.

This was clearly my Mistake and not a Mistake of the Launcher.
Therefore, I think that you do not have to take Action on this Point and the Issue can also be closed here.

I apologize for taking you Time

**Comment by bearinmindcat** [comment]
No problem at all! Just for reference, can you tell me what icon pack you're using and what apps aren't applying? This will be easier for me to figure out than guessing what the problem could be with icon parsing.

**Comment by ghost** [comment]
After it worked with my icon pack (which I created with Alembicons) without any problems, I compared again more accurately , which apps are not automatically assigned with an icon pack from the store. 


Meanwhile, I think the mistake there is not the launcher. 
I’ve determined that it’s always the same apps, no matter what icon pack I use (my whole open Source Appsfrom Droid-ify) 
For example, through the "Green Glass Orb" icon pack, I came to the conclusion that in the pack, for these apps simply no icon is explicitly stored and I have to manually select and assign the in each launcher.
With the Articons Packs from Droid-ify, almost all icons are automatically assigned. There are only three or four, very new or special apps (e.g. the AOSmium Browser "my camera app "SHiFT camera") that do not receive an icon. For everyone else, an icon is probably stored in the Articons Packs.

Therefore In the future and for other users, however, it would certainly be practical if then the possibility to set , icons for apps also directly manually from an icon pack. (as in the short video ,in the other topic)

I hope that I understood your contribution correctly and that it was also about the topic

**Comment by ghost** [comment]
What I noticed about the Topic "Folder & Talkback" after the Installation of v0.0.11 beta, if I put the Folders in the Dock, they are not recognized at all (so not even if I do not hide the label.)

I suspect that it is because there is generally no label displayed in the Dock (this is also a good Thing and should stay that Way )

But maybe you know another Way that Talkback can recognize the Folders in the Dock.

I’ll look for if another open source Launcher can do that and if I find one, I’ll let you know here. Maybe that could help you.

**Comment by ghost** [comment]
I have now searched for an open source Launcher where the Collaboration of "Folders in the Dock" and Talkback works.
It works well in the "Murine Launcher" (by alesimula)

In addition, the Folder Structure / Appearance could be interesting. The Launcher uses a small Window Display for Folders, which grows with the Number of Apps in the folder.

**Comment by bearinmindcat** [comment]
Going to close this since I completely changed the folder structure.

**Comment by UnknownViking** [comment]
I think the Issue was closed too early.

I have tested it and the Folders are only received by TalkBack if the Label is securable.

Even in the open Folder, the Behavior is still a bit strange.
The Apps and Labels are recognized separately and are also seperately selectable. This makes the Use rather cumbersome. 
In addition, the Labels cannot be made insectible within the Folders.

**Comment by bearinmindcat** [comment]
Alright I'll re-open. I realized I haven't actually touched this issue so I'll look further and get a long term workable fix for this hopefully!

**Comment by bearinmindcat** [comment]
FYI I've never used talkback in my life before and never even know this existed until now so this is all new to me. I'm not sure how it should work correctly, but I'll work on making this feel smooth with the folders, I see what you mean with the label being securable though (from the screenshot)

<img width="553" height="1306" alt="Image" src="https://github.com/user-attachments/assets/470612ce-4061-4904-bcef-3ea9bc24d450" />

**Comment by UnknownViking** [comment]
In Version 0.0.18 it works good 👍

Now i think also -  this Issue can be closed as completes

**Comment by bearinmindcat** [comment]
I still want to refine the talkback phrases and usage so I'll keep this open for a bit at least until I personally think everything relating to talkback feels smooth. There's a few kinks I want to iron out : )
